contrib/enfuse_pro_2.lua

```diff
@@ -0,0 +1,1050 @@
+--[[Enfuse professional plugin for darktable 2.2.X and 2.4.X
+
+  copyright (c) 2017, 2018  Holger Klemm (Original Linux-only version)
+  
+  darktable is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  
+  darktable is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[ Enfuse Pro 2 - cross-platform compatible
+Modified by: Kevin Ertel
+   
+ADDITIONAL SOFTWARE NEEDED FOR THIS SCRIPT
+* align_image_stack
+* enfuse
+* exiftool
+
+USAGE
+* require this file from your main luarc config file.
+
+This plugin will add the new export modul "fusion to DRI or DFF image".
+]]
```

Comment:
I would add that the first time you run darktable with it enabled you need to go to the lua options under preferences and set the executable paths, then restart darktable.

Reply:
Yea, I think I have that on my repo page, but adding it in the comments of hte file is defenitely the best way to go, you're right.

Reply:
Also, in regards to the spaces, this was a big trick for me. I found that if I quoted that paths in the arguments than windows would choke and not run AIS, enfuse, exiftool, etc properly. That's why I didn't quote them. I could do an if statement and only quote on mac and linux machines. Windows users would still have the limitation though. I will try again and see if for some reason quoting works under windows this time around, but that may be a limitation of this program for win. users, unless someone else knows a way around?

Additionally, I would love if there was a way to make the cmd windows that pop up on windows hidden or not have focus... I've asked around previously and didn't get much in response regarding that so I gave up.

Reply:
I meant quote the filenames of the image files, i.e. /usr/bin/align_image_stack '/path/to/file name1.cr2' '/path/to/file name2.cr2' '/path/to/file name3.cr2'.  If you don't quote this example align_image_stack sees the files as 
/path/to/file
name1.cr2
/path/to/file
name2.cr2
/path/to/file
name3.cr2

Reply:
Yea, the behavior is really odd on windows. Maybe you can help me solve what I am seeing. Here it is, with an example command and two scenarios below:
CMD = "C:\Program Files\Hugin\bin\align_image_stack.exe"  -i -C --distortion -g 5 -c 8 -t 3 --corr=0.9 --gpu -a "C:\Users\Kevin\AppData\Local\Temp\aligned_"  "C:\Users\Kevin\AppData\Local\Temp\_MG_5798.tif" "C:\Users\Kevin\AppData\Local\Temp\_MG_5800.tif" "C:\Users\Kevin\AppData\Local\Temp\_MG_5797.tif" "C:\Users\Kevin\AppData\Local\Temp\_MG_5799.tif"

NOTE: ALL paths are now quoted in the command including the executable path, output path, and source file paths.

1) Copy and paste the command into a cmd window and press enter, things run great, no complaints.
2) Modify enfuse_pro_2 script to build its command identical to above and it fails with the following error:
'C:\Program' is not recognized as an internal or external command, operable program or batch file.